pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = [
+    "setuptools<60.0.0",
+    "wheel",
+]
+
+[project]
+dependencies = [
+    "numpy>=1.19.5",
+    "pykokkos-base",
```

Comment:
Note that with `pip` these don't get installed in the user environment automatically, it is basically just metadata about what we need at runtime. Indeed, the idea is to only provide `pykokkos` into the user env without mutating it in any other way. I believe some other install tools may be able to use this data, though I haven't use them. The NumPy version is based loosely on [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html).

Reply:
I see, so there's no way to install pykokkos + pykokkos-base with a single pip command?

Reply:
You could add `pykokkos-base` to the `requires` section above and then do i.e., `pip install --no-build-isolation`, but `requires` is really only supposed to be used for genuine build *requirements*.

It still seems to me like the real problem is that `pykokkos-base` is a pain to install for people outside the project devs.